Foundation/NSNumber.swift

```diff
@@ -198,14 +198,20 @@ extension NSNumber : ExpressibleByFloatLiteral, ExpressibleByIntegerLiteral, Exp
 
 }
 
+@_silgen_name("_CFNumberGetType2")
```

Comment:
We really should not use silgen_name. Instead can you just put this in ForSwiftFoundationOnly.h?